### PR TITLE
Add references to PEPs 691, 700, and 714 to the simple repository API spec page

### PR DESCRIPTION
**Pull request by brettcannon**

### CODE REVIEW

File: source/key_projects.rst [hunk 1]
```diff
@@ -558,7 +558,7 @@ resolution by locally caching metadata about dependencies.
 pypiserver
 ==========
 
-`Docs <https://github.com/pypiserver/pypiserver/blob/master/README.rst>`__ |
+`Docs <https://github.com/pypiserver/pypiserver/blob/master/README.md>`__ |
```
Comment (webknjaz) [inline source/key_projects.rst:561]: This should probably just reference the repo page.

Comment (brettcannon) [reply source/key_projects.rst:561]: That's what I thought, too, but I didn't want to be too adventurous. 😅 I'll update the PR.

**Comment by brettcannon** [comment]
I also noticed some of the links in the file with the busted link point to `/en/latest/` links on Read the Docs which is probably not preferred in case the docs are translated.

File: source/key_projects.rst (hunk 1 above)
Comment (webknjaz) [reply source/key_projects.rst]: thanks!

**Pull request merged**